lib/ui/dart_runtime_hooks.cc

```diff
@@ -223,4 +226,51 @@ void ScheduleMicrotask(Dart_NativeArguments args) {
   UIDartState::Current()->ScheduleMicrotask(closure);
 }
 
+void LookupClosure(Dart_NativeArguments args) {
```

Comment:
This code will probably propagate error handles correctly, but I like to check Dart_Handle return values with DART_CHECK_VALID() unless there's some reason not to.

Reply:
Done.